docs/backends/remote.rst

```diff
@@ -1521,7 +1521,7 @@ Response:
 
 Get DomainInfo records for all domains in your backend.
 
--  Mandatory: no
+-  Mandatory: unless the zone cache has been disabled by setting zone-cache-refresh-interval to 0(not recommended for performance reasons)
```

Comment:
```suggestion
-  Mandatory: unless the zone cache has been disabled by setting :ref:`zone-cache-refresh-interval` to ``0`` (recommended for performance reasons)
```

Reply:
Uh, no, sorry, I somehow managed to get rid of the `not` while I just wanted to add a space :man_facepalming:

Reply:
I thin the :ref: should be :ref:`setting-zone-cache-refresh-interval`

Reply:
you can't, but you can make a new comment with an updated suggestion. The push works too, of course :-)